pilot/pkg/networking/core/cluster_builder.go

```diff
@@ -288,6 +288,7 @@ func (cb *ClusterBuilder) buildCluster(name string, discoveryType cluster.Cluste
 ) *clusterWrapper {
 	c := &cluster.Cluster{
 		Name:                 name,
+		AltStatName:          name + ";",
```

Comment:
Non-blocking question w.r.t. to the delimiter. Have you considered if it could be useful to have it as a constant value defined somewhere?

Reply:
So this altStatName would be overwritten again when user configures this https://github.com/istio/api/blob/master/mesh/v1alpha1/config.proto#L367?

It would be good if we can set the default vs user supplied one at one place so that it is easy to reason about? (Haven't looked at code in detail to see if this is possible)

Reply:
Nice catch; I wasn't even aware of that config option. I added a hold and will see if we can make this easier to read

Reply:
Looks like one of the customized alt stat names get set further down in that function (L347). I'll modify the BuildStatPrefix and BuildInboundStatPrefix functions to add the `;` so the user API stays consistent